src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1991,14 +2035,29 @@ RebalanceTableShardsBackground(RebalanceOptions *options, Oid shardReplicationMo
 						 move->targetNode->nodeId,
 						 quote_literal_cstr(shardTranferModeLabel));
 
-		BackgroundTask *task = ScheduleBackgroundTask(jobId, GetUserId(), buf.data,
-													  prevJobIdx, prevJobId);
-		prevJobId[prevMoveIndex] = task->taskid;
-		if (first)
+		int64 cid = GetColocationId(move);
```

Comment:
We usually don't abbreviate things.
```suggestion
		int64 colocationId= GetColocationId(move);
```